proto/cosmos/gov/v1/gov.proto

```diff
@@ -0,0 +1,191 @@
+syntax = "proto3";
+package cosmos.gov.v1;
+
+import "cosmos/base/v1beta1/coin.proto";
```

Comment:
I don't think it actually makes sense for a v1 proto package to depend on a v1beta1 package. I think we really need to bump coin to v1 first. Can we call this v1beta2 instead?

Reply:
Do you think that a change to v1 means that all dependencies should also be v1? i.e. all the modules step to v1 together?

Reply:
At least all dependencies should be v1 yes. Basically, calling this v1 would prevent us from updating its dependency to coin v1 which would be unfortunate...

Reply:
Yeah I guess this was a question I was asking myself: whether we bump versions on structs even when they haven't changed at all just because one of their dependents was bumped, or whether it's okay to have dependencies across versions

Reply:
If we want the former than I can either, as you say, just bump gov to v1beta2 or I can bump `coin.proto` to v1

Reply:
lets go with v1beta2, this PR is now blocking param rewrite work